scripts/char_lister.js

```diff
@@ -1,25 +1,28 @@
 // contributed by @junkie. Thanks!
-function (context, args) { // characters:#
+// edited by d3f4ult for more readability
+//this is called a low-dash parameter -- since we're not using context in this script *at all*, and we *need* to declare it, it is just an underscore
+//        ↓
+function (_, a) { // characters:""
 	//return on a failure
-	if (!args){
-		return{
+	if (!a){
+		return {
 			ok: false,
-			msg: "Define how many !characters! you want to display (The list starts at 01)."
+			msg: "Define how many `Ncharacters` you want to display (The list starts at 01)."
 		}
 	}
 	//iterates through a for loop, converting the numbers to characters
+	//note - for (let foo; foo<bar.length; foo++) syntax is less readable than a for...in loop - changed it
```

Comment:
The original one wasn't `a.characters.length`, so this is actually different functionality. It actually looks like the original was the intended form, in this case?